### PR TITLE
[virt_autotest] Add supportconfig as upload debug log

### PR DESCRIPTION
**Pull request by guoxuguang**
Add supportconfig as upload debug log

- Verification run: 
gi-guest_developing-on-host-developing-kvm
http://10.67.18.132/tests/127

### CODE REVIEW

**Comment by guoxuguang** [comment]
@alice-suse @waynechen55 @Julie-CAO please help me review this PR

File: tests/virt_autotest/virt_autotest_base.pm [hunk 1]
```diff
@@ -151,6 +151,15 @@ sub push_junit_log {
...
 
+sub upload_supportconfig_log {
+    my $datetab = script_output("date '+%Y%m%d%H%M%S'");
+    script_run("supportconfig -t /var/log -B supportconfig.$datetab", 600);
+    script_run("tar zcvfP /var/log/nts_supportconfig.$datetab.tar.gz /var/log/nts_supportconfig.$datetab");
+    upload_logs("/var/log/nts_supportconfig.$datetab.tar.gz");
+    script_run("rm -rf /var/log/nts_supportconfig.*");
+    save_screenshot;
+}
+
```
Comment (alice-suse) [inline tests/virt_autotest/virt_autotest_base.pm]: Please move this func to virt_utils.pm, which is the place for such utility functions.

Comment (guoxuguang) [reply tests/virt_autotest/virt_autotest_base.pm]: Will change code as your mentioned.

**Comment by Julie-CAO** [comment]
Your putting the support on the top level of the logs layout meets my flavor :)
but why remain the path of the supportconfig log, /var/log/...?

**Comment by guoxuguang** [comment]
> Your putting the support on the top level of the logs layout meets my flavor :)
> but why remain the path of the supportconfig log, /var/log/...?

Good question, because of based on the different SLES version, there was created the different supportconfig tar ball as default under /var/log dir.
For example:
There was created the nts_supportconfig.txz on SLES12SP4
There was created the nts_supportconfig.tbz on SLES15SP1 .. and so on
So, figure out that used -t option to just only save log files here, do not create tarball.

Also, /var/log as the default dir to save supportconfig log, so use -t option with /var/log 
to save supportconfig log under /var/log and do not create tarball here.

**Comment by Julie-CAO** [comment]
got the idea why put logs under /var/log/ and create a tar. I meant why upload supportconfig logs with the path /var/log/, it will be easier to read if the logs are listed in the tar directly, or we have to click twice to open two-level directories then read logs. but anyway, it is a minor issue.

**Comment by guoxuguang** [comment]
> got the idea why put logs under /var/log/ and create a tar. I meant why upload supportconfig logs with the path /var/log/, it will be easier to read if the logs are listed in the tar directly, or we have to click twice to open two-level directories then read logs. but anyway, it is a minor issue.

Good idea. 
Changed codes as your mentioned, which had been easier to read and all the logs were listed in the tar directly.

**Comment by guoxuguang** [comment]
Changes done, ready to be reviewed. If no more, hope someone can help merge, thanks!

**Pull request merged**